.husky/commit-msg

```diff
@@ -1,4 +1 @@
-#!/bin/sh
```

Comment:
These changes in `husky` are not related but I was getting this warning in the console when I was trying to commit new code:
![Screenshot 2024-10-04 at 11 12 22](https://github.com/user-attachments/assets/3427ee2b-ea56-4fc5-9745-66f7e14f40b9)